### DISCUSSION TITLE
Which modules are supported?

### DISCUSSION THREAD

**Discussion by ItsIgnacioPortal** [LSPosed/LSPatch#177] [Q&A]
I'm browsing https://modules.lsposed.org, but none of the modules that I've tried work :(

Is there a list to know what modules work with LSPatch? Do any modules work with LSPath?

According to [this thread](https://forum.xda-developers.com/t/xposed-app-5-0-alltrans-completely-translate-apps.3539878/), these modules work fine with LSPatch:
- [AllTrans](https://forum.xda-developers.com/t/xposed-app-5-0-alltrans-completely-translate-apps.3539878/): Replaces all text in an app in a language from one language to another at runtime.
- [Disable-FLAG-SECURE](https://apt.izzysoft.de/fdroid/index/apk/com.varuns2002.disable_flag_secure): This is an Xposed module to disable FLAG_SECURE, enabling screenshots, screen sharing and recording in apps that normally wouldn't allow it. It works with EdXposed as well as LSPosed.

**Comment by Programmerskynet** [comment]
### 

> [_****_](`[url](url)`)
